axelor-account/src/main/resources/views/Account.xml

```diff
@@ -102,7 +102,7 @@
           hideIf="accountType.technicalTypeSelect == 'view'" readonlyIf="isRegulatoryAccount"/>
         <field name="reconcileOk" widget="boolean-switch" colSpan="3"
           hideIf="accountType.technicalTypeSelect == 'view'"
-          readonlyIf="$isReadOnly || isRegulatoryAccount"/>
```

Comment:
Only keep the readonlyIf="isRegulatoryAccount"

Reply:
Solved